src/ResourceManager/AzureBackup/Commands.AzureBackup/packages.config

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.0.25-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.BackupServices" version="0.6.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.18.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net45" />
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.14.201151115" targetFramework="net45" />
+  <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.Management" version="4.1.1" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.Management.Scheduler" version="6.0.0" targetFramework="net45" />
+  <package id="Microsoft.WindowsAzure.Management.StorSimple" version="1.0.4-preview" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
```

Comment:
The idea here is to use the versions other projects are using, you can switch to newer one only if you need some new features. Otherwise, the installer will package into the new one with the old one and the download size will be bigger